### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by zhang96716** [MattScho/BikeShareIRL#2]
Hello, I'm very interested in your code. Is there a paper matching here

**Comment by MattScho** [comment]
Yes, this code is based on a journal paper my advisors and I have recently published, found here: http://jano.biemdas.com/archives/1209. I am currently expanding upon this work in my Master's thesis and will soon be posting that code as well.

**Comment by zhang96716** [comment]
Thank you for your reply, I've made some mistakes while running your code. Excuse me, 'exp erimentationSoftware.Agents.experiment ' and exp' erimentationSoftware.utilities.fileI0 'are these two files not uploaded?Can you send me a complete code? thank you

**Comment by MattScho** [comment]
Fixed! Let me know if you have any further questions or interesting findings.

**Comment by zhang96716** [comment]
In my run‘ parameterTest.py ’This file, the program reported an error.
Traceback (most recent call last):
File "C:/User/master/BikeShareIRL-master/experimentationSoftware/TestSuites/Static/ parameterTest.py ", line 28, in <module>
experiments.append (Experiment(title, PPO2(MlpPolicy, Monitor(BikeShareController(experiment),  "monitorFiles/" + title+"/"))))
TypeError: __ init__ () missing 1 required positional argument: 'resultsManager'
I really can't solve it. Please help me solve this mistake. Thank you very much.
